### ISSUE TITLE
About training detials in Table 3

### ISSUE THREAD

**Issue by tobran** [drboog/Lafite#15]
Your Lafite is a very important model. It is very exciting for me to find that Lafite can generate complex images of such high quality, showing amazing performance even compared with the large pre-trained models.
I am interested in some training details of Lafite.
Are the standard text-to-image Lafite models of different datasets in table 3 fine-tuned from pre-trained models?
If they are fine-tuned, how to pre-train the model for the different datasets? What is the pre-trained dataset? which is the training mode (language free or with gt) of pretraining?

Btw, thanks for sharing this wonderful work.

**Comment by drboog** [comment]
Hi, results from table 3 are obtained by training from scratch, there is no pre-training.

**Comment by drboog** [comment]
PS, actually pre-training + fine-tuning is not guaranteed to obtain better results in this model. I think this is dataset- or domain-dependent. 
In one of my previous experiment (not shown in the paper):
Pre-training with FFHQ + fine-tuning on MM-CelebA-HQ leads to better performance than training from scratch on MM-CelebA-HQ.
Pre-training on Google CC then fine-tuning on MS-COCO leads to worse results than training from scratch on MS-COCO.

**Comment by drboog** [comment]
Results from table 3 are trained with only ground-truth pairs.

**Comment by tobran** [comment]
I read other issues and find that the contrastive regularizer for D and G plays an important role in Lafite. My confusion is solved. Thank you very much for your explanation. It was great help.
